static/js/components/EducationDisplay.js

```diff
@@ -61,18 +61,18 @@ export default class EducationDisplay extends ProfileFormFields {
     );
     return (
       <Cell col={12} className="profile-form-row" key={index}>
-        <div className="basic-info">
+        <div className="profile-form-col">
           <div className="profile-row-name">
-            <span className="school-type">
-              { degree }
-            </span><br/>
-            { entry.school_name }
+            <div className="school-type">{ degree }</div>
+            <div className="school-name">{ entry.school_name }</div>
           </div>
+        </div>
+        <div className="profile-form-col education-top-spacing">
```

Comment:
also maybe `.education-top-spacing` -> `.education` and `.employment-top-spacing` -> `.employment`